manifests/p/ParallelsInternationalGmbH/ParallelsClient/20.1.25631/ParallelsInternationalGmbH.ParallelsClient.installer.yaml

```
@@ -0,0 +1,48 @@
+# Created using wingetcreate 1.9.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: ParallelsInternationalGmbH.ParallelsClient

```

```suggestion
PackageIdentifier: Parallels.Parallels
```

Please update file name and path to match this change.

---

Is it possible to use the package identifier I use because it has more details? Parallels has several products for Windows and identifier Parallels.Parallels does not even contain the uploaded product name